### PR TITLE
Update ABAC examples to use authenticated/unauthenticated groups

### PR DESCRIPTION
**Pull request by liggitt**
Doc update for https://github.com/kubernetes/kubernetes/pull/38968

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1992)

### CODE REVIEW

File: docs/admin/authorization.md [hunk 1]
```diff
@@ -506,7 +510,7 @@ An example request body:
     "resourceAttributes": {
       "namespace": "kittensandponies",
       "verb": "GET",
-      "group": "*",
+      "group": "unicorn.example.org",
```
Comment (deads2k) [inline docs/admin/authorization.md]: Cute, but its not obvious its a group. "unicorn-fan-club-members"?

Comment (liggitt) [inline docs/admin/authorization.md]: just matching later example

Comment (liggitt) [inline docs/admin/authorization.md]: and this is an API group, not a user group

Comment (deads2k) [inline docs/admin/authorization.md]: > and this is an API group, not a user group

Well, I think that went well.

**Comment by liggitt** [comment]
this is ready to merge (content applies to 1.5)

**Pull request merged**